src/widgets/HomePageWidgets/Hero/components/HeroSlider.tsx

```diff
@@ -0,0 +1,26 @@
+import { IHeroData } from '../model/types';
+
+export const HeroSlider = ({ hero }: IHeroData) => {
```

Comment:
Better to rename it to HeroSlide because it's actually just a single slide, but not a slider itself